lib/4_filter_numbers.php

```diff
@@ -8,8 +8,39 @@
 
 function filterNumbers($arr) {
   // Your code goes here.
+	$new_array = array();
+	foreach ($arr as $each_element) {
+		if ($each_element > 10){
+			//$new_array[] = $each_element;
+			array_push($new_array, $each_element);
+		}
+		return $new_array;
+	}
+
 }
 
 // Write your own "tests" below. Refer to the the examples from exercise #1.
+echo "\nTesting that 1 and 5 are removed from the $rubber_duck array. Should return true.\n";
```

Comment:
The project spec asked for numbers greater than or equal to 10 to be removed :) (But nice work on the reverse!)